packages/components/src/components/form-cancel-button/form-cancel-button.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+import Button, { TButtonProps } from '../button/button';
+
+type TFormCancelButton = Partial<TButtonProps> & {
+    className?: string;
+    is_absolute?: boolean;
+    is_center?: boolean;
+    is_disabled?: boolean;
+    label: string;
+    onCancel: () => void;
```

Comment:
Can the type or onCancel be MouseEvenetHandler?

Reply:
@likhith-deriv I believe this is a common callback, there is no need to type any eventhandlers.